### PR TITLE
Update for Deep Discovery Director Patch

### PR DESCRIPTION
**Pull request by chackco**
Thank you for your contribution! Some pointers to get it merged quickly:

For contributions in `software/`:

  - [ ] PR Title: Please use "Add <vendor/product name>" (instead of "Update README.md")
  - [ ] Status: please select a value from the status table at the top
  - [ ] Version: Status Vulnerable / Workaround? -> List vulnerable versions.
             Status Fix?                     -> List fixed versions.
  - [ ] Links: make sure you link to a public statement by the developer.
    Alternatively, include and link a file in the `software/vendor-statements/`
    directory
  - [ ] Please mind the sorting

### CODE REVIEW

File: software/README.md [hunk 1]
```diff
@@ -3111,7 +3111,7 @@ _Note: daily releases of this software list are listed, including CSV and JSON f
 | Trend Micro | Cloud One - Workload Secuity  |  | Not vuln | Not vuln | Not vuln | Not vuln | |[source](https://success.trendmicro.com/solution/000289940) |
 | Trend Micro | Cloud Sandbox  |  | Not vuln | Not vuln | Not vuln | Not vuln | |[source](https://success.trendmicro.com/solution/000289940) |
 | Trend Micro | Deep Discovery Analyzer  |  | Not vuln | Not vuln | Not vuln | Not vuln | |[source](https://success.trendmicro.com/solution/000289940) |
-| Trend Micro | Deep Discovery Director  |  | Not vuln | Workaround |  | | Patch expected 12/24 |[source](https://success.trendmicro.com/solution/000289940) |
+| Trend Micro | Deep Discovery Director  |  | Not vuln | Fix | Fix | Fix | Fix in 5.3 CP B1225 |[source](https://success.trendmicro.com/solution/000290055) |
```
Comment (tschmidtb51) [inline software/README.md:3114]:
```suggestion
| Trend Micro | Deep Discovery Director | 5.3 CP B1225 | Not vuln | Fix | Fix | Fix |  |[source](https://success.trendmicro.com/solution/000290055) |

**Review by tschmidtb51 — CHANGES_REQUESTED** [review]
hi @chackco, thanks for contributing to the software list. Can you have a look at my comments? Some changes are necessary before we can merge.

File: software/README.md (hunk 1 above)
Comment (chackco) [reply software/README.md:3114]: Yes, please merge using your suggestion

**Review by tschmidtb51 — APPROVED** [review]
Thank you for your contribution.

**Pull request merged**